pmpro-roles.php

```diff
@@ -387,7 +398,7 @@ public static function level_settings() {
 									}
 
 									if ( isset( $editable_roles[$custom_pmpro_role] ) ) { ?>
-										<div class="clickable">
+										<div class="pmpro_clickable" style="border-bottom-width: 4px;">
```

Comment:
Should we add another class and put this border CSS into https://github.com/strangerstudios/pmpro-roles/blob/dev/admin.css?